### ISSUE TITLE
[BUG] Uno.WinUI windows target is broken 

### ISSUE THREAD

**Issue by anpin** [PrismLibrary/Prism#2720]
### Description

Uno.WinUI target has some missing dependencies and nuget package `8.1.97` is not compatible with current version of  `Uno.WinUI` targeting `net6.0`

### Steps to Reproduce

1. clone `master`
2. build [PrismLibrary_Uno.sln](https://github.com/PrismLibrary/Prism/blob/master/PrismLibrary_Uno.sln)
3. It fails 

### Platform with bug

Uno Platform - WinUI

### Affected platforms

Windows

### Did you find any workaround?

I have changed conditionals in Prism.Uno.WinUI.csproj

### Relevant log output

```shell
....

D:\Prism\src\Uno\Prism.Uno\Prism.Uno.WinUI.csproj]
D:\Prism\src\Wpf\Prism.Wpf\Regions\RegionManager.cs(188,47): error CS0246: The type or namespace name
'DependencyObject' could not be found (are you missing a using directive or an assembly reference?) [D:\blav-app\module
s\Prism\src\Uno\Prism.Uno\Prism.Uno.WinUI.csproj]
D:\Prism\src\Wpf\Prism.Wpf\Regions\RegionManager.cs(201,45): error CS0246: The type or namespace name
'DependencyObject' could not be found (are you missing a using directive or an assembly reference?) [D:\blav-app\module
s\Prism\src\Uno\Prism.Uno\Prism.Uno.WinUI.csproj]
D:\Prism\src\Wpf\Prism.Wpf\Regions\RegionManager.cs(242,44): error CS0246: The type or namespace name
'DependencyObject' could not be found (are you missing a using directive or an assembly reference?) [D:\blav-app\module
s\Prism\src\Uno\Prism.Uno\Prism.Uno.WinUI.csproj]
D:\Prism\src\Wpf\Prism.Wpf\Regions\RegionManager.cs(51,32): error CS0246: The type or namespace name '
DependencyProperty' could not be found (are you missing a using directive or an assembly reference?) [D:\blav-app\modul
es\Prism\src\Uno\Prism.Uno\Prism.Uno.WinUI.csproj]
D:\Prism\src\Wpf\Prism.Wpf\Regions\RegionManager.cs(84,33): error CS0246: The type or namespace name '
DependencyProperty' could not be found (are you missing a using directive or an assembly reference?) [D:\blav-app\modul
es\Prism\src\Uno\Prism.Uno\Prism.Uno.WinUI.csproj]
D:\Prism\src\Wpf\Prism.Wpf\Regions\RegionManager.cs(140,32): error CS0246: The type or namespace name
'DependencyProperty' could not be found (are you missing a using directive or an assembly reference?) [D:\blav-app\modu
les\Prism\src\Uno\Prism.Uno\Prism.Uno.WinUI.csproj]
D:\Prism\src\Wpf\Prism.Wpf\Regions\RegionManager.cs(172,32): error CS0246: The type or namespace name
'DependencyProperty' could not be found (are you missing a using directive or an assembly reference?) [D:\blav-app\modu
les\Prism\src\Uno\Prism.Uno\Prism.Uno.WinUI.csproj]
    7 Warning(s)
    141 Error(s)
```

**Comment by anpin** [comment]
Found this issue after I hit this https://github.com/unoplatform/uno/issues/9396

**Comment by dansiegel** [comment]
This should have been fixed by #2822
